.travis.yml

```diff
@@ -1,15 +1,15 @@
 language: android
+jdk: oraclejdk8
```

Comment:
Why are we setting a specific JDK for the project? (I'm not a Travis expert, so I don't know what advantage this gives us)